hkm/templates/hkm/snippets/navigation.html

```diff
@@ -49,7 +49,7 @@
       {% if not user.profile.is_museum %}
       {% if user.is_authenticated %}
       <li class="list-item_user">
-        <a id="profile-dropdown" class="nav-user" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <a id="profile-dropdown" aria-label="{% trans 'Profile menu' %}" class="nav-user" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
Visual users will see a profile icon and I guess the user's name. Screen reader users should get the same information. It seems that currently only the icon is covered.

Reply:
I'll add user's name to `aria-label`